modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -38,7 +48,10 @@
   $disable_gc_alerts = false,
   $manage_repo = true,
   $open_firewall_from_all = false,
-  $backup_enabled = false
+  $backup_enabled = false,
+  $cors_enabled = false,
+  $cors_allow_headers = 'X-Requested-With, Content-Type, Content-Length',
+  $cors_allow_origin = 'http://app.quepid.com'
```

Comment:
Not exactly - we don't need to use production data for this tool, and I'm only using a free trial of Quepid at the moment - it's normally a paid service.

Thought I'd add in the host there because the default is to allow all hosts - if anyone does switches on CORS for something else they should probably specify the origin.